packages/discord-verify/src/lib/verify.ts

```diff
@@ -0,0 +1,125 @@
+import type {
+	Request,
+	SubtleCrypto,
+	SubtleCryptoImportKeyAlgorithm,
+} from "../types";
+
+export declare class TextEncoder {
+	constructor();
+	encode(input?: string): Uint8Array;
+}
+
+const encoder = new TextEncoder();
+
+/**
+ * Helper method that takes in a hex string and converts it to its binary representation.
+ * @param hex Hex string to convert to binary
+ * @returns The binary form of a hex string
+ */
+export function hexToBinary(hex: string | null) {
+	if (hex == null) {
```

Comment:
```suggestion
	if (hex === null) {
```

Reply:
double equality isn't needed for TS, but I think this works better for JS users